tap_linkedin_ads/client.py

```diff
@@ -228,23 +246,28 @@ def refresh_access_token(self):
 
         data = response.json()
         self.__access_token = data['access_token']
-
         # data['expires_in'] is an integer of seconds until the access_token expires.
         # Technically this self.__expires is inaccurate because it was true when LinkedIn generated the token, but
         # we receive and process that response some (very) small amount of time after it was true.
         self.__expires = datetime.utcnow() + timedelta(seconds=data['expires_in'])
 
+        self.write_access_token_to_config()
```

Comment:
I think we should only do this when we fetch a new token. Can this be moved inside the refresh_access_token() function?

Reply:
It is already inside the `refresh_access_token` function.

Reply:
Github UI collapsed and it read like it was in get_token_expires. My bad LOL